test/e2e_node/mirror_pod_test.go

```diff
@@ -378,6 +462,63 @@ spec:
 	return err
 }
 
+func createStaticPodWithActiveDeadlineSeconds(dir, name, namespace, image string, restart v1.RestartPolicy, activeDeadlineSeconds int64) error {
```

Comment:
one improvement here may be to return the pod spec from various createFoo methods similar to https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/standalone_test.go#L215 and write it to the file as a separate function. This way you can simply add the `activeDeadlineSeconds` to the basic pod instead of duplicating the whole functions.

I am ok if this will be a follow up refactoring, but also consider if you can do it as part of this PR

Reply:
done! Thanks for the suggestion.